Makefile

```diff
@@ -349,8 +356,10 @@ dist_export:
 	$(RSYNC) $(ADD-ON)/install.rdf.in $(DIST_EXPORT_DIR)
 	# jar.mn is used for Firefox build
 	rm -f $(DIST_EXPORT_DIR)/chrome.manifest
-	# Add the jar file.
+	rm -f $(DIST_EXPORT_DIR)/chrome/locale/chrome.manifest
+	# Add the jar files.
 	$(RSYNC) $(ADD-ON)/jar.mn $(DIST_EXPORT_DIR)
+	$(RSYNC) locale/jar.mn $(DIST_EXPORT_DIR)/chrome/locale
```

Comment:
The moz.build files are excluded from the export process. Generally they don't contain specific info about our files, so I didn't really want to include them in our repo.